source/_includes/puppet-collections/_puppet_collection_1_f20.md

```diff
@@ -0,0 +1,5 @@
+#### Fedora 20
+
+    sudo rpm -Uvh https://yum.puppetlabs.com/puppetlabs-release-pc1-fedora-20.noarch.rpm
+
+> **Note:** We do not ship packages for recent versions of Puppet on Fedora 20.
```

Comment:
A cutoff version would be useful, since as a reader I don't know what "recent" means.